.github/workflows/resource-indexer.yml

```diff
@@ -0,0 +1,40 @@
+name: rebuild the resource collection index
```

Comment:
Could someone remind me the best way to test this? (Once it is at the stage where I believe it'll work, of course.) Do I make a commit in master with an empty `.github/workflows/resource-indexer.yml`?

Reply:
cc @joverlee521 @victorlin @tsibley

Reply:
[This is still the workaround](https://github.com/nextstrain/docs.nextstrain.org/commit/aa6f75e103b83676345e3a0f7ac60291aa99e4da) I use, not sure if others have discovered new methods.

Reply:
Another workaround is to push a commit that adds the `pull_request` trigger. I just tested it in https://github.com/victorlin/github-test/pull/7.

Note that once it's added, it'll be run on every subsequent push to the PR, and should be dropped from the PR before merging if it's only for testing purposes. If you only want it to run once, you can push a commit to add it then push another one to revert it. Then before merging, just drop those two commits in a rebase.

Reply:
Ha! Wonderfully creative solutions. Thanks both :)